layers/+web/eww/packages.el

```diff
@@ -0,0 +1,125 @@
+;;; packages.el --- eww layer packages file for Spacemacs.
+;;
+;; Copyright (c) 2012-2017 Sylvain Benner & Contributors
+;;
+;; Author: Colton Kopsa <coljamkop@gmail.com>
+;; URL: https://github.com/syl20bnr/spacemacs
+;;
+;; This file is not part of GNU Emacs.
+;;
+;;; License: GPLv3
```

Comment:
You need to update the header.
See: https://github.com/syl20bnr/spacemacs/blob/develop/CONTRIBUTING.org#file-header